lib/feedjira/parser/podcast.rb

```diff
@@ -0,0 +1,82 @@
+require 'feedjira'
+require 'feedjira/parser/media_content'
+require 'feedjira/parser/media_group'
+require 'feedjira/parser/podcast_image'
+require 'feedjira/parser/podcast_item_enclosure'
+require 'feedjira/parser/podcast_item'
+
+module Feedjira
+  module Parser
+    class Podcast
```

Comment:
This is all in feeder as well, correct?  Should probably make this a shared module at some point.